src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -168,8 +168,8 @@ class ArchiveExplorer extends React.Component {
         }
 
         const sortedItems = itemList.sort((a = {}, b = {}) => {
-            if (!a[sortBy] || !b[sortBy]) {
-                return -1;
+            if (a[sortBy] === null || b[sortBy] === null) {
```

Comment:
Are any other non-string values possible? It seems like this change could cause us to throw an error on line 179.

Reply:
Maybe something like
```
            if (!a[sortBy] || !b[sortBy]) {
                return !a[sortBy] ? -1 : 1;
            }
```